app/src/main/java/dev/andrylat/app/CreditCardBrandDeterminer.java

```diff
@@ -0,0 +1,33 @@
+package dev.andrylat.app;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class CreditCardBrandDeterminer {
+    String determineCreditCardBrandByNumber(List<Integer> cardNumber) {
+        /*
+           for later improvements:
+
+           I could create an enum with all card brands,
+           and each card brand would have a method to validate
+           if the given number equals to the brand's possible number
+           the return type of the method would be boolean
+
+        */
+    	
+    	var brand = "";
```

Comment:
This is not according to rules.
![image](https://github.com/user-attachments/assets/c026c9e2-e93e-4ec8-9596-23c517646ca2)